Qualisys/QTM-Unity-Realtime-Streaming/Helpers/RTState.cs

```diff
@@ -131,6 +135,22 @@ public AnalogChannel GetAnalogChannel(string name)
             return null;
         }
 
+        
+        public ForceVector GetForcePlateVector(string name)
+        {
+            if (string.IsNullOrEmpty(name))
+                return null;
+
+            foreach (var force in forceVectors)
+            {
+                if (force.Name == name)
+                {
+                    return force;
+                }
+            }
+            return null;

```

Comment:
Suggestion: Could be replaced with LINQ statement .FirstOrDefault.

Reply:
> Suggestion: Could be replaced with LINQ statement .FirstOrDefault.

We usually refrain from using LINQ because of the amount of garbage it creates.